test/slider-test.jsx

```diff
@@ -19,147 +25,149 @@ function testKeys(slider, tests) {
 
 const newSlider = props => new Slider({ ...Slider.defaultProps, ...props });
 
-describeWithDOM('<Slider />', () => {
-  describe('render', () => {
```

Comment:
why do we need to remove the "render" describe? if you restore it, the diff will be much smaller and thus more readable.